tasks/25_rot13-encoder/rot13-encoder.js

```diff
@@ -0,0 +1,18 @@
+// Make an empty array, we know that the charCode range for
+// uppercased letters is between 65 and 90.
+// We swap the charCodes by adding or substracting 13 from original charCode.
+
+export const rot13Encoder = ( string ) => {
+  let result = [];
+
+  for ( let i = 0; i <= string.length; i++ ) {
+    if ( string.charCodeAt(i) >= 65 && string.charCodeAt(i) <= 77 ) {
```

Comment:
it is better to make a variable for `string.charCodeAt(i)` then call this method a lot times.

Reply:
Refactored